engine/physics/src/box2d/Box2D/Dynamics/b2World.h

```diff
@@ -241,6 +247,8 @@ class b2World
 
 	b2Vec2 m_gravity;
 	bool m_allowSleep;
+	int m_velocityIteration;
+	int m_positionIteration;
```

Comment:
These aren't needed, since these are passed in as parameters to the b2World::Step() function.

Reply:
That's right, I will remove them.